login.js

```diff
@@ -121,11 +113,18 @@ define('login',
             if (opt.register) {
                 fxa_url = utils.urlparams(fxa_url, {action: 'signup'});
             }
-            fxa_popup = window.open(
-                fxa_url,
-                'fxa',
-                'scrollbars=yes,width=' + w + ',height=' + h +
-                ',left=' + i[0] + ',top=' + i[1]);
+
+            if (opt.popupWindow) {
+                console.log('Changing location of supplied window to', fxa_url);
+                fxa_popup = opt.popupWindow;
+                fxa_popup.location = fxa_url;
+            } else {
+                console.log('Opening new login window');
+                fxa_popup = utils.openWindow({url: fxa_url});
+            }
+
+            // This denotes the popup should be kept around for re-use.
+            retainPopup = opt.retainPopup || false;
```

Comment:
if ``opt.popupWindow`` is specified then I feel like this should default to true